From 0061cf481739e444400118604c3c7f7deb912230 Mon Sep 17 00:00:00 2001
From: Jammy Huang <jammy_huang@aspeedtech.com>
Date: Sun, 26 Oct 2025 18:45:23 +0800
Subject: [PATCH] board: ibex_ast2700: Enable pcie settings for all 27xx

Remove efuse check.

Signed-off-by: Jammy Huang <jammy_huang@aspeedtech.com>
Change-Id: Ieaafddf8c0959c93b3595554d699222f1624062b
(cherry picked from commit d6c5c53e69af77dae95f68ed0d128ca01267f91b)
Signed-off-by: Vasanthakumar K <vasanthakumark@ami.com>
---
 board/aspeed/ibex_ast2700/ibex_ast2700.c | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/board/aspeed/ibex_ast2700/ibex_ast2700.c b/board/aspeed/ibex_ast2700/ibex_ast2700.c
index a544e0abb499..509c53b3aec3 100644
--- a/board/aspeed/ibex_ast2700/ibex_ast2700.c
+++ b/board/aspeed/ibex_ast2700/ibex_ast2700.c
@@ -139,7 +139,6 @@ static int pci_init(void)
 	int nodeoffset;
 	ofnode node;
 	struct ast2700_scu0 *scu;
-	u8 efuse;
 
 	/* find the offset of compatible node */
 	nodeoffset = fdt_node_offset_by_compatible(gd->fdt_blob, -1,
@@ -160,12 +159,6 @@ static int pci_init(void)
 		return 0;
 	}
 
-	efuse = FIELD_GET(SCU_CPU_REVISION_ID_EFUSE, scu->chip_id1);
-	if (efuse == 2) {
-		debug("%s: 2720 has no PCIE\n", __func__);
-		return 0;
-	}
-
 	/* cpu-die pcie node 1 */
 	// setup preset for plda2
 	writel(0x12600000, (void *)ASPEED_PLDA2_PRESET0);
-- 
2.34.1

